examples/basic-storagevolumes/tidb-cluster.yaml

```diff
@@ -0,0 +1,58 @@
+# IT IS NOT SUITABLE FOR PRODUCTION USE.
+# This YAML describes a basic TiDB cluster with minimum resource requirements,
+# which should be able to run in any Kubernetes cluster with storage support.
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v5.1.0
+  timezone: UTC
+  pvReclaimPolicy: Retain
+  enableDynamicConfiguration: true
+  configUpdateStrategy: RollingUpdate
+  discovery: {}
+  pd:
+    baseImage: pingcap/pd
+    replicas: 1
+    # if storageClassName is not set, the default Storage Class of the Kubernetes cluster will be used
+    # storageClassName: local-storage
+    requests:
+      storage: "1Gi"
+    config: {}
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 1
+    # if storageClassName is not set, the default Storage Class of the Kubernetes cluster will be used
+    # storageClassName: local-storage
+    requests:
+      storage: "1Gi"
+    config:
+      storage:
+        # In basic examples, we set this to avoid using too much storage.
+        reserve-space: "0MB"
+      rocksdb:
+        # In basic examples, we set this to avoid the following error in some Kubernetes clusters:
+        # "the maximum number of open file descriptors is too small, got 1024, expect greater or equal to 82920"
+        max-open-files: 256
+      raftdb:
+        max-open-files: 256
+    separateRocksDBLog: true
+    separateRaftLog: true
+    raftLogVolumeName: raftlog
+    rocksDBLogVolumeName: rocksdblog
+    storageVolumes:
+    - name: raftlog
+      storageClassName: local-storage
+      storageSize: 1Gi
+      mountPath: /var/lib/raftlog
```

Comment:
The raft log path should also be set in the `config` field.

Reply:
Raft log path will be the result of concating mountPath and `raft/LOG`, shall this satisfied?

Reply:
Yes, it's OK.

Reply:
No, you have to set the log path to the new mount path. @mianhk